### ISSUE TITLE
The location list does not retain its position when the screen is rotated on large screen devices.

### ISSUE THREAD

**Issue by Netocon** [breezy-weather/breezy-weather#683]
### Steps to reproduce

Close the location list and rotate the screen

### Expected behavior

The Location list should remain closed or only expand on in landscape mode (closing again when on portrait mode)

### Actual behavior

The location list opens regardless of whether it was closed

### Weather source used

AccuWeather

### Breezy Weather version

4.6.14

### Android version

Android 12

### Device

Redmi Pad

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I read the [Frequently Asked Questions](https://github.com/breezy-weather/breezy-weather/blob/main/HELP.md) document and followed instructions if relevant.
- [X] I have updated the app to **[version 4.6.14 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance), unless my issue only happens with a specific language
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
The behavior on a tablet is the following:

If location list is empty:
- Always open by default to prompt the user to add a location

If location list is not empty:
- Closed if in portrait
- Open if in landscape

I.e. if you're in portrait you probably want your drawer to be closed, but when you switch to landscape, have it open. You can still change that behavior manually.

Note that we don't save the opening status between rotations/sessions, so if you open the drawer in portrait mode, then switch to landscape, then switch back to portrait, the location list will be closed.


TL;DR: There is no such feature as "position retaining", closing as "not a bug".
